source/supported-hardware/qsp.rst

```diff
@@ -0,0 +1,70 @@
+.. _getting-started_qsp:
+
+
+QSP Simics Platform
+-----------------------
+
```

Comment:
Please add a line about the HW platform - x85 + ICH10

Reply:
I guess it is x58.

Reply:
yes, thanks.